frontend-public/src/selectors/varianceSelectors.js

```diff
@@ -0,0 +1,75 @@
+import * as varianceReducer from "@/reducers/varianceReducer";
+import { createSelector } from "reselect";
+import * as Strings from "@/constants/strings";
+import { createLabelHash, createDropDownList } from "@/utils/helpers";
+
+export const { getMineVariances, getComplianceCodes, getVarianceStatusOptions } = varianceReducer;
+
+export const getVarianceApplications = createSelector(
+  [getMineVariances],
+  (variances) =>
+    variances.filter(
+      ({ variance_application_status_code }) =>
+        variance_application_status_code !== Strings.VARIANCE_APPROVED_CODE
```

Comment:
Are we okay with grouping denied, not-applicable, and in-review variances together under "variance applications?"

Based on the name, I'd expect it to just match against the `REV` status code

Reply:
That is also what I thought, but according to T, these are still considered applications, they are just no longer in the "In-Review" stage.